r2/r2/controllers/api.py

```diff
@@ -752,6 +754,9 @@ def POST_leavecontributor(self, container):
         """
         if container and container.is_contributor(c.user):
             container.remove_contributor(c.user)
+            if container.is_moderator_with_perms(c.user, 'access'):
```

Comment:
@Deimos The reasoning I had was that when mods remove anyone or themselves from being a mod, a mod action was created; and there's also a mod action made when removing themselves as a contributor if they remove themselves from the /about/contributor page (as that itself is done via `POST_unfriend` and not `POST_leavecontributor`)